<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResampleDialog</class>
 <widget class="QDialog" name="ResampleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resample ROI - ITK-SNAP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Segmentation ROI Resolution:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,1,0,1,0,1">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>8</number>
         </property>
         <property name="margin">
          <number>4</number>
         </property>
         <item row="4" column="3">
          <widget class="QDoubleSpinBox" name="outInputZvox">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>x:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>y:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>z:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QDoubleSpinBox" name="inOutVoxelYvox">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>voxels</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../Resources/SNAPResources.qrc">:/root/media-playback-start-4.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="outInputYvox">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="3" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Input ROI</string>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="QDoubleSpinBox" name="inOutVoxelXvox">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>mm/voxel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="7">
          <widget class="QDoubleSpinBox" name="inOutVoxelZvox">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>mm/voxel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>voxels</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5" colspan="3" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Resampled ROI</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="4" column="6">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5" colspan="3">
          <widget class="QCheckBox" name="chkAspect">
           <property name="text">
            <string>Fixed voxel aspect ratio</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="outInputXsize">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="outInputYsize">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="outInputXvox">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QSpinBox" name="inOutVoxelZsize">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QSpinBox" name="inOutVoxelYsize">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QSpinBox" name="inOutVoxelXsize">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="outInputZsize">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Interpolation:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="inInterpolationMode"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnPreset">
        <property name="text">
         <string>Presets</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionSuper2">
   <property name="text">
    <string>Supersample by 2</string>
   </property>
  </action>
  <action name="actionSub2">
   <property name="text">
    <string>Subsample by 2</string>
   </property>
  </action>
  <action name="actionSuperIso">
   <property name="text">
    <string>Supersample to Isotropic</string>
   </property>
  </action>
  <action name="actionSubIso">
   <property name="text">
    <string>Subsample to Isotropic</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inOutVoxelXsize</tabstop>
  <tabstop>inOutVoxelXvox</tabstop>
  <tabstop>inOutVoxelYsize</tabstop>
  <tabstop>inOutVoxelYvox</tabstop>
  <tabstop>inOutVoxelZsize</tabstop>
  <tabstop>inOutVoxelZvox</tabstop>
  <tabstop>chkAspect</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/SNAPResources.qrc"/>
 </resources>
 <connections/>
</ui>
